### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by pboling**
one character typo in Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -184,7 +184,7 @@ a = {x:{0=>1}}
 b = {x:[1]}
 
 diff = HashDiff.diff(a, b, :array_path => true)
-diff.should == [["~", [:a], [1], {0=>1}]]
+diff.should == [["~", [:x], [1], {0=>1}]]
```
Comment (krzysiek1507) [inline README.md]: It should be `[["~", [:x], {0=>1}, [1]]]`

Comment (pboling) [reply README.md]: Good point, it was even more wrong 🥇

Comment (pboling) [inline README.md]:
```suggestion
diff.should == [["~", [:x], {0=>1}, [1]]]
```

**Comment by liufengyun** [comment]
Thanks a lot @pboling !

**Pull request merged**